### PR TITLE
Update trainer's list and improve the footer

### PR DESCRIPTION
**Pull request by tinavahedi**
CHANGES:

    1. Add missing trainer's information to the main slide.
    2. Fix a subtle issue in the footer that would cause a visual inconsistency in mobile view.

This PR introduces improvements and informational update without breaking existing functionality.

### CODE REVIEW

File: src/components/Footer.astro [hunk 1]
```diff
@@ -40,7 +42,7 @@ let phone       = "+982122063869";
                     </ul>
                 </div>
             </div>
-            <div class="md:grid md:grid-cols-2 md:gap-8 mr-8">
+            <div class="md:grid gap-10 mr-8 ">
```
Comment (kfv) [inline src/components/Footer.astro]: Remove the redundant trailing space.

File: src/components/Footer.astro [hunk 2]
```diff
@@ -7,9 +7,9 @@ let phone       = "+982122063869";
 ---
 
 <footer id="footer" class="relative flex py-24 justify-center bg-black" dir="rtl" >
-    <div class="justify-center items-center md:grid-cols-4 grid-cols-3 grid-rows-2">
-        <div class="mt-10 grid grid-cols-2 md:gap-8">
-            <div class="md:grid md:grid-cols-2 md:gap-8 mr-8">
+    <div class="justify-center items-center  md:grid-cols-4 grid-rows-2">
```
Comment (kfv) [inline src/components/Footer.astro]: Remove the redundant space before `md:...`.

File: src/components/Footer.astro
Comment (kfv) [inline src/components/Footer.astro:1]: Remove all redundant white-spaces and update the repo, please.

Comment (tinavahedi) [reply src/components/Footer.astro:1]: Done.

**Review by kfv — APPROVED** [review]
Thanks!

**Pull request merged**